ros_gz_shims/ros_ign/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <!-- TODO: Make this a metapackage, see
+       https://github.com/ros2/ros2/issues/408 -->
+  <name>ros_ign</name>
+  <version>0.0.1</version>
+  <description>Shim meta-package to redirect to <a href="https://github.com/gazebosim/ros_gz">ros_gz</a>.</description>
+  <maintainer email="brandon@openrobotics.org">Brandon Ong</maintainer>
+  <license>Apache 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <exec_depend>ros_gz</exec_depend>
+
+  <exec_depend>ros_ign_bridge</exec_depend>
+  <exec_depend>ros_ign_gazebo</exec_depend>
+  <exec_depend>ros_ign_gazebo_demos</exec_depend>
+  <exec_depend>ros_ign_image</exec_depend>
+  <!-- See https://github.com/gazebosim/ros_gz/issues/40 -->
+  <!--exec_depend>ros_gz_point_cloud</exec_depend-->
```

Comment:
I was intending to make this look like a metapackage, but your approach makes much more sense, I'll make the edit :raised_hands:

Reply:
Edit: Actually I realized in this case the other dependencies (`ros_ign_`) are for the shims, and are there so they're pulled in (by rosdep) if you just have `ros_ign`.

Ala metapackages! Editing `ros_gz` won't have any effect because `ros_gz` will be using `ros_gz_`

Reply:
Ahhhh good point, sorry for the noise